mlos_bench/mlos_bench/launcher.py

```diff
@@ -168,32 +183,40 @@ def service(self) -> Service:
     def _parse_args(
         parser: argparse.ArgumentParser,
         argv: Optional[List[str]],
-    ) -> Tuple[argparse.Namespace, List[str]]:
+    ) -> Tuple[argparse.Namespace, List[str], List[str]]:
         """Parse the command line arguments."""
+        path_args = []
+
         parser.add_argument(
             "--config",
             required=False,
-            help="Main JSON5 configuration file. Its keys are the same as the"
-            + " command line options and can be overridden by the latter.\n"
-            + "\n"
-            + " See the `mlos_bench/config/` tree at https://github.com/microsoft/MLOS/ "
-            + " for additional config examples for this and other arguments.",
+            help=(
+                "Main JSON5 configuration file. Its keys are the same as the "
+                "command line options and can be overridden by the latter.\n"
+                "\n"
+                "See the `mlos_bench/config/` tree at https://github.com/microsoft/MLOS/ "
+                "for additional config examples for this and other arguments."
+            ),
         )
+        path_args.append("config")
 
         parser.add_argument(
             "--log_file",
             "--log-file",
             required=False,
             help="Path to the log file. Use stdout if omitted.",
         )
+        path_args.append("log_file")
```

Comment:
Maybe define
```python
class PathArgsTracker:

    def __init__(self, parser: argparse.ArgumentParser):
        self.parser = parser
        self.path_args = []

    def __call__(self, *args, **kwargs) -> None:
        self.path_args.append(self.parser.add_argument(*args, **kwargs).dest)
```
So that then we can write
```python
        path_args_tracker = PathArgsTracker(parser)

        path_args_tracker(
            "--log_file",
            "--log-file",
            required=False,
            help="Path to the log file. Use stdout if omitted.",
        )
```
and access `self.path_args` at the end

alternatively, we can write it simply as
```python
        path_args = []
        track_path_args = lambda arg: path_args.append(arg.dest)

        track_path_args(parser.add_argument(
            "--log_file",
            "--log-file",
            required=False,
            help="Path to the log file. Use stdout if omitted.",
        ))
```